Documentation/install/upgrade.rst

```diff
@@ -141,15 +137,13 @@ configuration options for each minor version.
 
 .. include:: ../gettingstarted/k8s-install-download-release.rst
 
-Generate the required YAML file and deploy it:
+Deploy Cilium release via Helm:
 
-.. code:: bash
+.. parsed-literal::
 
-   helm template cilium \
-     --namespace kube-system \
-     --set config.enabled=false \
-     > cilium.yaml
-   kubectl apply -f cilium.yaml
+   helm upgrade cilium |CHART_RELEASE| \\
+     --namespace kube-system \\
+     --set config.enabled=false
```

Comment:
What's the appropriate way to roll this back? Does a standard `kubectl rollout undo daemonset/cilium -n kube-system` still work or should we update the instructions below?

Reply:
I have updates this section to use `helm rollback` instead of `kubectl rollout undo`.

Reply:
I discussed this with some other developers, there's some concern that only documenting usage of helm upgrade has too many unknowns in terms of existing user base and helm version deployment.

It'd be good to have separate tabs to allow users to opt into `helm upgrade` or just generate the YAML and apply via `kubectl`.

I'm happy to either merge the PR as-is then prepare the changes per this feedback, or if you'd like to make such a change that can work too. Let me know.

Reply:
Added tabs with `helm` and `kubectl` versions of the commands.